README.md

```diff
@@ -1,41 +1,150 @@
 
-## Standalone Usage
+  * `se_create_symbolic_link_privilege`
+    define which users are allowed to create symbolic links
 
-This Compliance Profile requires [InSpec](https://github.com/chef/inspec) for execution:
+  * `se_deny_network_logon_right`
+    define which users are not allowed to access this computer from the network
+
+  * `se_deny_remote_interactive_logon_right`
+    define which users are not allowed to log on through Remote Desktop Services
+
+  * `se_enable_delegation_privilege`
+    define which users are allowed to enable computer and user accounts to be trusted
+
+  * `se_impersonate_privilege`
+    define which users are allowed to impersonate a client after authentication
+
+  * `se_load_driver_privilege`
+    define which users are allowed to impersonate a client after authentication
+
+  * `se_batch_logon_right`
+    define which users are allowed to log on as a batch job
+
+  * `se_security_privilege`
+    define which users are allowed to manage auditing and security logs
+
+  * `se_assign_primary_token_privilege`
+    define which users are allowed to replace a process level token
+
+  * `se_restore_privilege`
+    define which users are allowed to restore files and directories
+
+  * `hklm_null_session_pipes`
+    define named pipes that can be accessed anonymously
+
+## Usage
+
+InSpec makes it easy to run your tests wherever you need. More options listed here: [InSpec cli](http://inspec.io/docs/reference/cli/)
 
 ```
+# run profile locally and directly from Github
+$ inspec exec https://github.com/dev-sec/windows-baseline
+
+# run profile locally
 $ git clone https://github.com/dev-sec/windows-baseline
 $ inspec exec windows-baseline
-```
 
-You can also execute the profile directly from Github:
+# run profile on remote host via WinRM
+inspec exec windows-baseline -t winrm://<ip-address>:5985 --user=<username> --password=<password>
 
+# run profile on remote host via WinRM and define attribute value
+inspec exec windows-baseline -t winrm://<ip-address>:5985 --user=<username> --password=<password> --attrs sample_attributes.yml
+
+# run profile direct from inspec supermarket
+inspec supermarket exec dev-sec/windows-baseline -t winrm://<ip-address>:5985 --user=<username> --password=<password>
 ```
-$ inspec exec https://github.com/dev-sec/windows-baseline
 
-# run test on remote windows host on WinRM
-$ inspec exec test.rb -t winrm://Administrator@windowshost --password 'your-password'
+### Run individual controls
+
+In order to verify individual controls, just provide the control ids to InSpec:
+
+```
+inspec exec windows-baseline --controls 'windows-001'
 ```
 
-## License and Author
+## ToDo
 
-* Copyright 2015-2016, Chef Software, Inc
-* Copyright 2016, The Hardening Framework Team
+- adjust the inspec attributes according to the profile (Member Server or Domain Controller), because for the Domain Controller some attributes are different from a Memeber Server
```

Comment:
It seems like this has been done above? Is that still a todo?

Reply:
It is still a small todo, because a couple of controls (around 10) need some other attribute values

Reply:
@chris-rock My pleasure!